src/datetime/addHours.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * To add hours to a datetime.
+ * @param {Date} dateFrom
+ * @param {number} hours
+ * @return Date
+ */
+
+export const addHours = (dateFrom: Date, hours: number): Date => {
```

Comment:
This function's implementation is almost identical to `addMilliseconds` (except for the conversion from hours to milliseconds). Consider calling `addMilliseconds` to avoid duplicated code.

Reply:
I think that new implementation is straightforward.